### ISSUE TITLE
My output image converted is blurry

### ISSUE THREAD

**Issue by prise6** [deepfakes/faceswap-playground#80]
Hi everyone,

Thank you for this amazing code.
I'm trying to train a model to swap 2 faces. 
At the end, after every training, and a a convert on sample, face B on face A is blurry. 
I'm quite new in this field, so i wanted to know few things:

* How many images for each face ? 
* Which loss score can be accepted ?
* Does the size of image input has a big impact?

Here is my example : 

* face A : ~900 img
* face B : ~257 img
* batch size : 64
* epoch : 3000
* final loss : [#02999] loss_A: 0.05209, loss_B: 0.04585

And one output example of convert with theses options : -c Masked -M rect -S

![Example](http://upload.timfaitsoncinema.fr/p/2018-03/5a9ab162.jpg)

How can i improve the output ? Does someone have similar issue and was able to improve it ?
Do i need to change size input of the neural net ?

Thank you for your time.
Feel free to ask me further explanations.

**Comment by torzdf** [comment]
I'd raise your epoch to default and just stop the process when the loss levels are acceptable.

You want to be aiming for loss of < 0.02 for both A and B, so the results your are getting look about correct for 0.05.

Everything else looks ok. Just train longer.

**Comment by prise6** [comment]
Thank you! I appreciate
